docs/ocis/getting-started/demo-users.md

```diff
@@ -16,13 +16,13 @@ To skip the generation of demo users, run the inital setup step with an addition
 
 Following users are available in the demo set:
 
-| username  | password      | email                 | role                | groups                                                                  |
-| --------- | ------------- | --------------------- | ------------------- | ----------------------------------------------------------------------- |
-| admin     | admin         | admin@example.org     | admin               | users                                                                   |
-| einstein  | relativity    | einstein@example.org  | user                | users, philosophy-haters, physics-lovers, sailing-lovers, violin-haters |
-| marie     | radioactivity | marie@example.org     | user                | users, physics-lovers, polonium-lovers, radium-lovers                   |
-| moss      | vista         | moss@example.org      | admin               | users                                                                   |
-| richard   | superfluidity | richard@example.org   | user                | users, philosophy-haters, physics-lovers, quantum-lovers                |
-| katherine | gemini        | katherine@example.org | user, space-manager | users, sailing-lovers, physics-lovers, quantum-lovers                   |
+| username  | password      | email                 | role        | groups                                                                  |
+| --------- | ------------- | --------------------- | ----------- | ----------------------------------------------------------------------- |
+| admin     | admin         | admin@example.org     | admin       | users                                                                   |
+| einstein  | relativity    | einstein@example.org  | user        | users, philosophy-haters, physics-lovers, sailing-lovers, violin-haters |
+| marie     | radioactivity | marie@example.org     | user        | users, physics-lovers, polonium-lovers, radium-lovers                   |
+| moss      | vista         | moss@example.org      | admin       | users                                                                   |
+| richard   | superfluidity | richard@example.org   | user        | users, philosophy-haters, physics-lovers, quantum-lovers                |
+| katherine | gemini        | katherine@example.org | space admin | users, sailing-lovers, physics-lovers, quantum-lovers                   |
```

Comment:
She is not a `user` any more? She used to be before? Why is this happening?

Reply:
users cannot be part of two roles... so admin incorporates users permissions and so needs a space manager (see https://github.com/owncloud/ocis/issues/3302)

Reply:
Ok. Commented on the ticket. Thanks!